Source/WebCore/page/Quirks.cpp

```diff
@@ -564,9 +565,9 @@ bool Quirks::shouldMakeTouchEventNonCancelableForTarget(EventTarget* target) con
     auto host = m_document->topDocument().url().host();
 
     if (equalLettersIgnoringASCIICase(host, "www.youtube.com"_s)) {
```

Comment:
This needs to be a follow-up PR, but this makes no sense. _host_ is canonicalized by the URL parser so we can compare for equality.

Reply:
If by "makes no sense" you mean "is slightly suboptimal", then agreed.